### PR TITLE
Add (apparently) required about text to issue links

### PR DESCRIPTION
**Pull request by sagikazarmark**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -2,12 +2,16 @@ blank_issues_enabled: false
...
     url: https://github.com/go-kit/kit/discussions/new?category=q-a
+    about: Ask questions and discuss topics with other community members
 
   - name: Website
     url: https://gokit.io/
+    about: Check the website for details, examples and documentation
 
   - name: Reference
     url: https://pkg.go.dev/github.com/go-kit/kit
+    about: Check the Go code reference
```
Comment (peterbourgon) [inline .github/ISSUE_TEMPLATE/config.yml:13]:
```suggestion
    about: Go kit package documentation
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -2,12 +2,16 @@ blank_issues_enabled: false
 contact_links:
   - name: Ask a question
     url: https://github.com/go-kit/kit/discussions/new?category=q-a
+    about: Ask questions and discuss topics with other community members
 
   - name: Website
     url: https://gokit.io/
+    about: Check the website for details, examples and documentation
```
Comment (peterbourgon) [inline .github/ISSUE_TEMPLATE/config.yml:9]:
```suggestion
    about: Project overview, examples, frequently asked questions, etc.
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 3]
```diff
@@ -2,12 +2,16 @@ blank_issues_enabled: false
 contact_links:
   - name: Ask a question
     url: https://github.com/go-kit/kit/discussions/new?category=q-a
+    about: Ask questions and discuss topics with other community members
```
Comment (peterbourgon) [inline .github/ISSUE_TEMPLATE/config.yml:5]:
```suggestion
    about: Questions and discussions with the Go kit community
```

File: .github/ISSUE_TEMPLATE/config.yml [hunk 4]
```diff
@@ -2,12 +2,16 @@ blank_issues_enabled: false
...
     url: https://gokit.io/
+    about: Check the website for details, examples and documentation
 
   - name: Reference
     url: https://pkg.go.dev/github.com/go-kit/kit
+    about: Check the Go code reference
 
   - name: Slack channel
     url: https://gophers.slack.com/messages/go-kit
+    about: Ask questions here in realtime
```
Comment (peterbourgon) [inline .github/ISSUE_TEMPLATE/config.yml:17]:
```suggestion
    about: Real-time discussions and Q&A
```

**Review by peterbourgon — APPROVED** [review]
Just some nits

**Comment by sagikazarmark** [comment]
Cool, solved: 

Képernyőfotó 2021-07-21 - 0 01 13

**Pull request merged**